### ISSUE TITLE
[Accessibility][Screen Reader] [fd-input] Screen reader cannot get the required infomartion

### ISSUE THREAD

**Issue by Meggie886** [SAP/fundamental-ngx#13456]
### Describe your proposal

Read the input is marked as required.
Read the information tip of the input.

### Which versions of Angular and Fundamental Library for Angular are affected?

latest

### Steps to Reproduce (for bugs)

1. Open input example [https://fundamental-ngx.netlify.app/#/core/input](https://fundamental-ngx.netlify.app/#/core/input) 
2. Locate  the required example 

<img width="2272" height="752" alt="Image" src="https://github.com/user-attachments/assets/51a1b2ec-a3cf-4e73-85b0-8b47e51da42e" />
3. Open the screen reader
<img width="2716" height="1688" alt="Image" src="https://github.com/user-attachments/assets/4fccf2b4-cf8a-4ee1-a5e5-f98bb17f9fb6" />

<img width="2326" height="1388" alt="Image" src="https://github.com/user-attachments/assets/5bf29592-cae6-4d17-a41f-73c59e552aba" />

4. Locate [https://fundamental-ngx.netlify.app/#/core/input](https://fundamental-ngx.netlify.app/#/core/input) 

<img width="2304" height="1136" alt="Image" src="https://github.com/user-attachments/assets/3e6e847c-82d3-47a0-80dd-67c700ffbad7" />

5. Open the screen reader,The error information tip cannot be read. Post by
 [https://github.com/SAP/fundamental-ngx/issues/13455](https://github.com/SAP/fundamental-ngx/issues/13455)

<img width="1079" height="409" alt="Image" src="https://github.com/user-attachments/assets/6ba538e1-7dfd-4d6c-8cd0-89b58ca11dc5" />

### Declaration

- [x] I'm not disclosing any internal or sensitive information.
